.circleci/config.yml

```diff
@@ -115,6 +115,7 @@ commands:
             gcloud firebase test android run --type instrumentation \
               --app <<parameters.app_apk_path>> \
               --test <<parameters.test_apk_path>> \
+              --device model=panther,version=33,locale=en,orientation=portrait \
```

Comment:
Tests started failing on the Pixel5,30 we were using by default before... I tried this and they started passing again, maybe gcloud had some issues with the device 🤷‍♂️ . For now,  I think this is ok.

Reply:
If we're testing on 1 api level I think a recent one makes sense!